WBooks/MainTabBar/LibraryTab.swift

```diff
@@ -21,6 +21,14 @@ class LibraryTab: UITabBarController {
         return .lightContent
     }
     
+    override open var shouldAutorotate: Bool {
+        return false
+    }
+    
+    override var supportedInterfaceOrientations: UIInterfaceOrientationMask {
```

Comment:
If all your app is portrait, you can set it in Target's configuration
![image](https://user-images.githubusercontent.com/42350557/59061070-89bb9f00-8870-11e9-8e5f-81678146553c.png)